rpc_jobs/rpc_maas.yml

```diff
@@ -19,6 +45,8 @@
       - master:
           SLAVE_TYPE: "nodepool-maas-ubuntu-xenial-p2-15"
           TRIGGER_PR_PHRASE_ONLY: true
+      - rocky:
+          SLAVE_TYPE: "nodepool-maas-ubuntu-xenial-g1-8"
```

Comment:
Can these tests pass in Phobos? If so, this can just be ``nodepool-ubuntu-xenial-g1-8``.